controllers/topics.js

```diff
@@ -274,16 +276,12 @@ router.post('/', async (req, res, next) => {
 
 	api(req).post('/lessons/', {
 		json: data, // TODO: sanitize
-	}).then(() => {
+	}).then((lesson) => {
 		if (req.body.referrer) {
 			res.redirect(`${(req.headers.origin)}/${req.body.referrer}`);
 		}
-		res.redirect(
-			context === 'courses'
-				? `/courses/${req.params.courseId
-				}${req.query.courseGroup ? `/groups/${req.query.courseGroup}` : '/?activeTab=topics'}`
-				: `/teams/${req.params.teamId}/?activeTab=topics`,
-		);
+
+		res.redirect(`${(req.headers.origin)}/courses/${req.params.courseId}/topics/${lesson._id}/edit`);
```

Comment:
How it handle lessons in coursegroups?

https://default-main.cd.dbildungscloud.dev/courses/62eb686155c2e66cb2c5d4ba/topics/62f226c8c239f2001d5d28f4?courseGroup=62f226b7c239f2001d5d2747

Teacher login

Reply:
The '?courseGroup='-Param doesn't makes a difference on the edit page. On the detail page it adds the hierarchy navigation on top. But for consistency reasons I added it with 2a15438009c584db1a7c61dd6dc227eb99879335
FYI @csp175